src/engraving/layout/layoutchords.cpp

```diff
@@ -1247,3 +1249,75 @@ void LayoutChords::repositionGraceNotesAfter(Segment* segment)
         }
     }
 }
+
+void LayoutChords::updateLineAttachPoints(Measure* measure)
+{
+    for (Segment& s : measure->segments()) {
+        if (!s.isChordRestType()) {
+            continue;
+        }
+        for (EngravingItem* e : s.elist()) {
+            if (!e || !e->isChord()) {
+                continue;
+            }
+            Chord* c = toChord(e);
+            for (Note* n : c->notes()) {
+                n->lineAttachPoints().clear();
+            }
+            for (Chord* ch : c->graceNotes()) {
+                for (Note* n : ch->notes()) {
+                    n->lineAttachPoints().clear();
+                }
+            }
+        }
+    }
+    for (Segment& s : measure->segments()) {
+        if (!s.isChordRestType()) {
+            continue;
+        }
+        for (EngravingItem* e : s.elist()) {
+            if (!e || !e->isChord()) {
+                continue;
+            }
+            Chord* c = toChord(e);
+            doUpdateLineAttachPoints(c);
```

Comment:
is that a right name, @mike-spa ?
the method doUpdateLineAttachPoints doesn't change lineAttachPoints, so it's confusing

Reply:
added comments to clarify!